core/domain/exp_domain.py

```diff
@@ -2811,6 +2792,48 @@ def _convert_states_v12_dict_to_v13_dict(cls, states_dict):
                 state_dict['interaction']['solution'] = None
         return states_dict
 
+    @classmethod
+    def _convert_states_v13_dict_to_v14_dict(cls, states_dict):
+        """Converts from version 13 to 14. Version 14 adds
+        audio translations to feedback, hints, and solutions.
+        """
+        for state_dict in states_dict.values():
+            if (state_dict['interaction']['default_outcome'] is not None and
```

Comment:
Perhaps this (up to line 2812) is better structured as:

```
            if state_dict['interaction']['default_outcome'] is not None:
                old_feedback_list = state_dict['interaction']['default_outcome']['feedback']
                default_feedback_html = (
                    old_feedback_list[0] if len(old_feedback_list) > 0 else '')
                state_dict['interaction']['default_outcome']['feedback'] = {
                    'html': default_feedback_html,
                    'audio_translations': {}
                }
```

Reply:
done